modules/special-rules/pages/supreme-commander.adoc

```diff
@@ -9,10 +9,18 @@
 Where your unit has the {supreme-commander} ability:
 
 * All of your detachments that have their HQ unit within 30 cm of the {supreme-commander} unit gain the {stubborn} ability.
-* If this specific unit fights in a close combat or contributes its Firepower to a firefight then you may re-roll the die that decides the outcome of that close combat or firelight.
-If you re-roll the die then you must abide by the second result.
+* If the unit contributes its Assault value to a close combat or contributes its Firepower value to a firefight then you may re-roll a single die when making the combat resolution roll.
+If you re-roll a die then you must abide by the second roll.
```

Comment:
I think it'd still make best sense to stick with singular here, because that is the default and the optional alternate method explains itself anyway.
Also, I intend to separate 'close combat' and 'firefight' clearly throughout, and try "Assault resolution" as the more general term. So, I put it that way in this suggestion and I'll tweak others in similar fashion and see how it looks overall.

```suggestion
* If the unit contributes its Assault value to a close combat, or contributes its Firepower value to a firefight, then you may re-roll the die for Assault resolution.
If you re-roll the die then you must abide by the second roll.
```